### DISCUSSION TITLE
How do i add my own classes to the rendered markdown

### DISCUSSION THREAD

**Discussion by samgurung** [spatie/laravel-markdown#82] [General]
As my question says i want to add some classes to the rendered markdown. Specifically i am using tailwind css and would like to add some tailwind classes for margin and padding to the fenced code in my markdown. Will appreciate any ideas y'all have

**Comment by searsandrew** [comment]
I tend to add a parent class to the markdown tag, then in my app.css I use Tailwinds @apply tag and style that way. Hope that helps. 

For example; `<x-markdown class="markdown">`

app.css
```
.markdown ul li { @apply list-disc list-inside; }
.markdown ol li { @apply list-decimal list-outside ml-6 pl-2 mb-2; }
.markdown ol li:last-child { @apply mb-0; }
.markdown ol li ul li { @apply list-disc list-outside; }
.markdown h2 { @apply text-xl font-semibold text-slate-800 mb-1; }
.markdown p { @apply mb-2; }
```
